bindata/network/dhcp/dhcp-daemon.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: dhcp-daemon
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the containernetworking plugins DHCP daemon on each node.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: dhcp-daemon
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: dhcp-daemon
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      initContainers:
+      - name: cni-plugins-supported
+        image: {{.CNIPluginsSupportedImage}}
+        command: ["/bin/sh"]
+        args: ["-c", "rm -f /var/run/cni/dhcp.sock"]
+        securityContext:
+          privileged: true
```

Comment:
even assuming the `dhcp-daemon` container needs to be privileged, why does the `initContainer` need to be privileged? (also, needs a better `name`)